Code/Editor/EditorViewportSettings.h

```diff
@@ -20,6 +20,7 @@ namespace SandboxEditor
 {
     using AngleSnappingChangedEvent = AZ::Event<bool>;
     using GridSnappingChangedEvent = AZ::Event<bool>;
+    using PerspectiveChangedEvent = AZ::Event<>;
```

Comment:
update: I think it would be good to add a float parameter here to pass through the new fov value after it's changed (similar to how the other events work)